app/fileManager.js

```diff
@@ -60,16 +73,63 @@ export default function(state, emitter) {
     emitter.emit('render');
   }
 
+  /*
+
+  function updateDloads() {
+    emitter.emit('updateDloads');
+  }
+*/
+
+  /*const dlimitArray = [];
+  const fileIdArray = [];
+  function ifRerender(file) {
+    if (fileIdArray.indexOf(file.id) === -1) {
+      const fileDetails = {
+        fileId: file.id,
+        dlimit: file.dlimit,
+        dtotal: file.dtotal
+      };
+      fileIdArray.push(file.id);
+      dlimitArray.push(fileDetails);
+      return false;
+    } else {
+      for (const fileDetail of dlimitArray) {
+        if (file.id === fileDetail.fileId) {
+          if (
+            file.dlimit !== fileDetail.dlimit ||
+            file.dtotal !== fileDetail.dtotal
+          ) {
+            fileDetail.dtotal = file.dtotal;
+            fileDetail.dlimit = file.dlimit;
+            return true;
+          } else {
+            return false;
+          }
+        }
+      }
+    }
+  }*/
```

Comment:
Please remove the commented out code

Reply:
yep sure I'll make all these changes :+1: